### ISSUE TITLE
How to automate a function in a smart contract?

### ISSUE THREAD

**Issue by qnxdev** [jklepatch/eattheblocks#159]
Despite its name, ‘smart’ contracts in Ethereum are not self-executing. You need an external source (either human or machine) to call the smart contract and execute its code. 

But projects like TIME wonderland has already automated rebase rewards. I wonder how is it possible.

Can someone explain?

Thanks in advance

**Comment by moesaleh** [comment]
Using Chainlink Keepers Node, you can run automated jobs in a decentralized manner

**Comment by Kali-Decoder** [comment]
Yes You Can automate smart contracts by the help of external package or external smart contract that helps to automate your smart contract functions that is Chainlink Keepers Oracle system Developed by chain link.
Chainlink Keepers provide two function one for register your automate function and other one for self called function to automate your smart contract function and it takes gasFees also in terms Of LINK Token of chain link. hope it will Help you.

Happy Codding !!!
